integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/azure/azurecloudblob/action/AzureCloudBlobClientActions.java

```diff
@@ -91,43 +87,39 @@ private void createCloudBlobContainer(String containerName) {
     }
 
     private String getContainerName(String baseLocation) {
-        final String containerName = baseLocation.substring(baseLocation.lastIndexOf("//") + 1, baseLocation.lastIndexOf("@"));
-        LOGGER.info("Container Name: {} at Base Location: {}", containerName, baseLocation);
+        String containerName = baseLocation.substring(baseLocation.lastIndexOf("//") + 1, baseLocation.lastIndexOf("@"));
```

Comment:
I would extract this to a method and give it a meaningful name (e.g. extract containerNameFromBaseLocation)

Reply:
This has been already extracted in a separate method. So I cannot see any beneficial outcome to create another additional method for getting the Azure storage container name:
- based on the given parameter, that is `baseLocation` or
- with no parameter that means `azureProperties.getCloudStorage().getBaseLocation()` based on the `application.yaml`.